app/views/layouts/application.slim

```diff
@@ -9,4 +9,5 @@ html
   body
     #notice
     = link_to 'User', users_path
-    = yield 
\ No newline at end of file
+    = yield
+    
```

Comment:
if you want to add new line please be attentive about spaces in this line, because git thinks it isn't new line ;)